### DISCUSSION TITLE
Is Gutenberg production ready ?

### DISCUSSION THREAD

**Discussion by amarinediary** [WordPress/gutenberg#37714] [Q&A] [answered]
A question that is pretty straight forward, Is Gutenberg production ready in correlation to WordPress ?

I'm starting a new project and noticed that some blocks behaviors which were dysfunctional early 2021 (`template` and `template_lock` options from the `register_post_type` function) are still dysfunctional. Should I expect the whole system do be revamped ?

Many thanks in advance,

**Comment by aristath** [comment, marked as answer]
Gutenberg is the playground where we work on features that will eventually make their way to WordPress Core.
The plugin is as stable as it can be, but at the same time, it's still a playground!
If you are building a production website you don't need to use Gutenberg. Use WordPress Core instead. Everything that has been battle-tested and proved to be 100% stable in Gutenberg, gets ported to core. New features get added in each major WordPress release.
Some things in Gutenberg cannot be considered "final", and they do get revamped & refactored regularly as we search for an optimal solution that will eventually make its way in WordPress.
